cdci_data_analysis/plugins/dummy_instrument/data_server_dispatcher.py

```diff
@@ -239,4 +239,13 @@ def check_query_roles(self, roles, par_dic):
                     'general': 'general role is needed for p>50',
                     'unige-hpc-full': 'unige-hpc-full role is needed for p>50 as well'
                 }
+        if 'p_list' in par_dic.keys():
+            p_list = par_dic.get('p_list', '')
+            if float(p_list[0]) > 50:
+                results['authorization'] = 'general' and 'unige-hpc-full' in roles
```

Comment:
this is not tested, right?

Reply:
I didn't since the rules are exactly the same as for the `p` parameter

Reply:
I thought about it, and since is not tested, and there is actual no need for it, I will remove it from here the roles check.